kafka-rest/src/main/java/io/confluent/kafkarest/ProducerMetricsRegistry.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest;
+
+public class ProducerMetricsRegistry {
+
+  public static final String GROUP_NAME = "produce-api-metrics";
+
+  // sensor names
+  public static final String REQUEST_SENSOR = "request-sensor";
+  public static final String REQUEST_SIZE_SENSOR = "request-size-sensor";
+  public static final String RESPONSE_SENSOR = "response-sensor";
+  public static final String RECORD_ERROR_SENSOR = "record-error-sensor";
+  public static final String REQUEST_LATENCY_SENSOR = "request-latency-sensor";
+
+  // request
+  public static final String REQUEST_RATE = "request-rate";
+  public static final String REQUEST_RATE_DOC = "The average number of requests sent per second.";
+
+  public static final String REQUEST_SIZE_AVG = "request-size-avg";
+  public static final String REQUEST_SIZE_AVG_DOC = "The average request size in bytes.";
+
+  public static final String REQUEST_COUNT_WINDOWED = "request-count-windowed";
+  public static final String REQUEST_COUNT_WINDOWED_DOC =
+      "The total number of requests sent within the given window.";
+
+  // response
+  public static final String RESPONSE_SEND_RATE = "response-rate";
+  public static final String RESPONSE_SEND_RATE_DOC =
+      "The average number of response sent per second.";
+
+  public static final String RESPONSE_COUNT_WINDOWED = "response-count-windowed";
+  public static final String RESPONSE_COUNT_WINDOWED_DOC =
+      "The total number of responses sent in the given window.";
+
+  // errors
+  public static final String ERROR_COUNT_WINDOWED = "error-count-windowed";
+  public static final String ERROR_COUNT_WINDOWED_DOC =
+      "The total number of record sends that resulted in errors in the given window.";
+
+  public static final String RECORD_ERROR_RATE = "record-error-rate";
+  public static final String RECORD_ERROR_RATE_DOC =
+      "The average per-second number of record sends that resulted in errors.";
+
+  // latency
+  public static final String REQUEST_LATENCY_AVG = "request-latency-avg";
+  public static final String REQUEST_LATENCY_AVG_DOC = "The average request latency";
+
+  public static final String REQUEST_LATENCY_MAX = "request-latency-max";
+  public static final String REQUEST_LATENCY_MAX_DOC = "The max request latency";
```

Comment:
That's also a problem for my "favorite" rate metrics, but here too the mention of average, max, and percentiles brings up the question of the aggregation window. Am I correct in thinking that latency is tracked as Kafka Metrics `SampledStat` and therefore it uses some window that's not the whole lifetime of the metrics sensor? Can you please document that here?

Reply:
I've added a comment to this effect, the default window size is 30s

Reply:
I mostly meant the `_DOC` constants, as that's what brought the question of the aggregation window to my attention. If these are actually exposed to end users, we should probably include the clarification there; if not, your comment seems perfectly fine to me.